.github/workflows/build_vagrant_win.yml

```
@@ -0,0 +1,49 @@
+name: Vagrant Playbook Checker
+
+on:
+  pull_request:
+    paths:
+    - .github/workflows/build_vagrant_win.yml
+    - ansible/playbooks/AdoptOpenJDK_Windows_Playbook/**
+    branches:         
+    - master
+
+jobs:
+  win2016:
+    name: Windows 2016
+    runs-on: macos-latest
+    steps:
+
+    - uses: actions/checkout@v2
+
+    - name: Install dependencies
+      run: |
+        brew install ansible
+        brew install --cask virtualbox vagrant
+        vagrant plugin install vagrant-disksize
+
+    - name: Setup Vagrant VM
+      run: |
+        cd ansible
+        ln -sf Vagrantfile.Win2016 Vagrantfile
+        vagrant up
+        rm -f playbooks/AdoptOpenJDK_Windows_Playbook/hosts.win
+        # 5986 refers to the winrm_ssl port on the guest
+        # See: https://github.com/AdoptOpenJDK/openjdk-infrastructure/issues/1504#issuecomment-672930832
+        vagrantPort=$(vagrant port | awk '/5986/ { print $4 }')
+        echo "[127.0.0.1]:$vagrantPort" >> playbooks/AdoptOpenJDK_Windows_Playbook/hosts.win
+        echo "This is the content of hosts.win : " && cat playbooks/AdoptOpenJDK_Windows_Playbook/hosts.win
+        sed -i -e "s/.*hosts:.*/- hosts: all/g" playbooks/AdoptOpenJDK_Windows_Playbook/main.yml
+        # Uncomments and sets the ansible_password to 'vagrant', in adoptopenjdk_variables.yml
+        sed -i'' -e "s/.*ansible_password.*/ansible_password: vagrant/g" playbooks/AdoptOpenJDK_Windows_Playbook/group_vars/all/adoptopenjdk_variables.yml
+        # If "credssp" isn't found in adoptopenjdk_variables.yml
+        if ! grep -q "credssp" playbooks/AdoptOpenJDK_Windows_Playbook/group_vars/all/adoptopenjdk_variables.yml;
+        then
+          # Add the "ansible_winrm_transport" to adoptopenjdk_variables.yml
+          echo -e "\nansible_winrm_transport: credssp" >> playbooks/AdoptOpenJDK_Windows_Playbook/group_vars/all/adoptopenjdk_variables.yml
+        fi
+
+    - name: Run Ansible Playbook
+      run: |
+        cd ansible
+        ansible-playbook -i playbooks/AdoptOpenJDK_Windows_Playbook/hosts.win -u vagrant -b --skip-tags adoptopenjdk playbooks/AdoptOpenJDK_Windows_Playbook/main.yml
```

```suggestion
        ansible-playbook -i playbooks/AdoptOpenJDK_Windows_Playbook/hosts.win -u administrator --skip-tags adoptopenjdk playbooks/AdoptOpenJDK_Windows_Playbook/main.yml
```
Had a quick check, the Administrator account has the same password as the vagrant account (i.e. `vagrant`), and stops the need for the `-b` tag, that's causing an issue :-)